eg/Classes/WindowsVersion.py

```diff
@@ -0,0 +1,221 @@
+#
+# EventGhost is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free
+# Software Foundation, either version 2 of the License, or (at your option)
+# any later version.
+#
+# EventGhost is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with EventGhost. If not, see <http://www.gnu.org/licenses/>.
+
+import platform
+
+
+class WindowsVersion(object):
+    """
+    A convenience module for checking the windows version.
+
+    Functions:
+        IsXP()
+        IsVista()
+        Is7()
+        Is8()
+        Is80()
+        Is81()
+        Is10()
+
+    Comparison:
+        eg.WindowsVersion [<|<=|==|!=|>=|>] "[XP|Vista|7|8|80|81|10]"
+            XP:    Windows XP
+                   Windows Server 2003
+                   Windows 2003 R2
+                   Windows Tablet PC
+                   Windows Media Center Edition 2002
+                   Windows Media Center Edition 2004
+                   Windows Media Center Edition 2005
+            Vista: Windows Vista
+                   Windows Server 2008
+            7:     Windows 7
+                   Windows Server 2008 R2
+            8:     (80 and 81)
+            80:    Windows 8
+                   Windows Server 2012
+                   Windows RT
+            81:    Windows 8.1
+                   Windows 2012 R2
+                   Windows RT 8.1
+            10:    Windows 10
+                   Windows Server 2016
+    """
+
+    __docsort__ = (
+        "IsXP, IsVista, Is7, Is8, Is80, Is81, Is10"
+    )
+
+    WIN_10 = [10, 0]
+    WIN_81 = [6, 3]
+    WIN_80 = [6, 2]
+    WIN_7 = [6, 1]
+    WIN_VISTA = [6, 0]
+    WIN_XP64 = [5, 2]
+    WIN_XP32 = [5, 1]
+
+    class WindowsVersionError(Exception):
+        def __init__(self, msg):
+            self.msg = msg
+
+        def __str__(self):
+            return (
+                "'{0}' is not in the list of supported values:\n"
+                "XP, Vista, 7, 8, 80, 81, 10\n".format(self.msg)
+            )
+
+    def __init__(self):
+        ver = platform.version().split('.')
+        self.THIS = [int(ver[0]), int(ver[1])]
+
+    @staticmethod
+    def check_os_value(val):
+        if val not in ['XP', 'Vista', '7', '8', '80', '81', '10']:
+            raise WindowsVersion.WindowsVersionError(val)
+
+    def __eq__(self, other):
+        self.check_os_value(other)
+        return getattr(self, 'Is'+other)()
+
+    def __ne__(self, other):
+        self.check_os_value(other)
+        return not getattr(self, 'Is' + other)()
+
+    def __gt__(self, other):
+        self.check_os_value(other)
+        if other == 'XP':
+            return self.THIS >self. WIN_XP32 and self.THIS >self. WIN_XP64
```

Comment:
Formatting:

    return self.THIS > self.WIN_XP32 and self.THIS > self.WIN_XP64